### PR TITLE
Wind model and test

### PR DESCRIPTION
**Pull request by dguittet**
Originally gmlc-dispatches/#8

### CODE REVIEW

File: dispatches/models/renewables_case/wind_power.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    def build(self):
+        """Building model
+
+        Args:
+            None
+        Returns:
+            None
+        """
+        super(WindpowerData, self).build()
```
Comment (andrewlee94) [inline dispatches/models/renewables_case/wind_power.py:79]: Minor comment: IDAES only supports Python 3, so we can just do `super().build()`

**Review by andrewlee94 — APPROVED** [review]
Looks OK to me. I only had one minor comment.

File: dispatches/models/renewables_case/wind_power.py (hunk 1 above)
Comment (dguittet) [reply dispatches/models/renewables_case/wind_power.py]: Fixed

**Review by ksbeattie — CHANGES_REQUESTED** [review]
The annotate_source script should be run on these files to add the copyright/license headers.

**Review by andrewlee94 — APPROVED** [review]
Looks good.

**Comment by jghouse88** [comment]
@dguittet can you please check and resolve conflicts?

**Comment by dguittet** [comment]
@jghouse88 Thanks, fixed up conflicts. Ready for re-review.

**Review by jghouse88 — APPROVED** [review]
LGTM

**Review by andrewlee94 — APPROVED** [review]

**Review by ksbeattie — APPROVED** [review]
LGTM

**Pull request merged**